<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 7pt &quot;Arial&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="FileName">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>50</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Browse">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>BROWSE</string>
    </property>
   </widget>
   <widget class="QLabel" name="id3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>200</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string> id:</string>
    </property>
   </widget>
   <widget class="QLabel" name="id2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>80</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string> id:</string>
    </property>
   </widget>
   <widget class="QLabel" name="id1">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>80</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string> id:</string>
    </property>
   </widget>
   <widget class="QLabel" name="image3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>220</y>
      <width>60</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../docs/empty.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="image4">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>220</y>
      <width>60</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../docs/empty.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="image1">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>100</y>
      <width>60</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../docs/empty.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="image2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>100</y>
      <width>60</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../docs/empty.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="id4">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>200</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string> id:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textResult">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>261</width>
      <height>101</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:7pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun'; font-size:9pt;&quot;&gt;:) message box&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="inputimage">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>60</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../docs/empty.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="similar">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>50</y>
      <width>241</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>SHOW SIMILAR PHOTOS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>320</y>
      <width>281</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(178, 203, 255);</string>
    </property>
    <property name="text">
     <string>Go To Searching Mode</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>569</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuinput">
    <property name="title">
     <string>input</string>
    </property>
   </widget>
   <addaction name="menuinput"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
